### ISSUE TITLE
Quick link to source pages are broken

### ISSUE THREAD

**Issue by ccarouge** [ACCESS-Hive/access-hive.github.io#589]
At the top right of each page, there are icons to either view or edit one of the pages. They are supposed to link to the page on GitHub but they are now broken.

This was broken with this change: https://github.com/ACCESS-Hive/access-hive.github.io/commit/3918fcda7c0b7da72f2501d83542556fa5a2b447 that removes the details of the GitHub repository URL in `mkdocs.yml`.

I have no idea if we want to bring back the GitHub repository details or if we want to remove the broken icons. To remove the icons, we need to remove these options in `mkdocs.yml`:

```yaml
features:
    - content.action.view
    - content.action.edit
```

**Comment by atteggiani** [comment]
I would not leave those icons there. 
I don't think they serve a lot of purpose with the current Development system/workflow.

**Comment by KAUR1984** [comment]
Thanks for raising this issue @ccarouge, and providing precise details as the issue description. We are considering to remove these icons for now, however, we would come back to this in the future in case they are needed in the future.

**Comment by KAUR1984** [comment]
Moving it to in progress for removing the icons from the Hive website.

**Comment by atteggiani** [comment]
Fixed in #605
